taskRequests/details/style.css

```diff
@@ -32,13 +40,13 @@ body {
   max-width: 1440px;
```

Comment:
Why do we have max-width for the body this will look absurd  on large screens

Reply:
I am not sure why this was added as this is a leftover task by Tanishq

Reply:
can we fix it in this PR? yes but not a blocker for this PR

Reply:
will try to address this in future pull requests since removing them is resulting in significant CSS modifications.